desktop/release-notes.md

```diff
@@ -23,13 +23,98 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
+## Docker Desktop 4.11.0
+2022-07-28
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Added two new extensions ([vcluster](https://hub.docker.com/extensions/loftsh/vcluster-dd-extension) and [PGAdmin4](https://hub.docker.com/extensions/mochoa/pgadmin4-docker-extension)) to the Extensions Marketplace.
+- Ability to sort extensions has been added to the Extensions Marketplace.
+- No more feedback nagging. Thanks to all your feedback, we identified a bug that was causing some users to be asked for feedback too frequently. You should now only be asked for feedback twice a year.
+- Custom theme settings for Docker Desktop allow you to specify dark or light mode for Docker Desktop independent of your device settings. Fixes [docker/for-win#12747](https://github.com/docker/for-win/issues/12747)
+- Add new flag for Windows installer, `--no-windows-containers` to disable Windows containers integration.
+
+### Upgrades
+- [Docker Compose v2.7.0](https://github.com/docker/compose/releases/tag/v2.7.0)
+- [Docker Compose "Cloud Integrations" v1.0.28](https://github.com/docker/compose-cli/releases/tag/v1.0.28)
+- [Kubernetes v1.24.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.2)
+- [Go 1.18.4](https://github.com/golang/go/releases/tag/go1.18.4)
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Adds the Container / Compose icon as well as the exposed port(s) / exit code to the Containers screen.
+- Updated Docker theme palette colours values to match design system.
+- Improved an error message from `docker login` if Registry Access Management is blocking the Docker engine's access to Docker Hub.
+- Increase throughput between the Host and Docker, for example increasing performance of `docker cp`.
+- Collecting diagnostics takes less time to complete.
+- Selecting/unselecting a compose app on the containers overview will select/unselect all its containers.
+- Show tag names on the container overview image column.
+- Added search decorations to the terminal's scrollbar so that matches outside the viewport are visible.
+- Fixed issue on search which doesn't work well on containers page [docker/for-win#12828](https://github.com/docker/for-win/issues/12828).
+- Fixed issue on infinite loading on volume screen [docker/for-win#12789](https://github.com/docker/for-win/issues/12789).
+- Fixed a problem in the Container UI where resizing or hiding columns didn't work. Fixes [docker/for-mac#6391](https://github.com/docker/for-mac/issues/6391).
+- Fixed a bug where the state of installing, updating, or uninstalling multiple extensions at once was lost when leaving the Marketplace screen.
+- Fixed issue where compose version in the about page would only get updated from v2 to v1 after restart.
+- Fixed an issue where user cannot not see log view because their underlying hardware didn't support WebGL2 rendering. Fixes [docker/for-win#12825](https://github.com/docker/for-win/issues/12825).
```

Comment:
```suggestion
- Fixed an issue where users cannot see the log view because their underlying hardware didn't support WebGL2 rendering. Fixes [docker/for-win#12825](https://github.com/docker/for-win/issues/12825).
```